.drone.yml

```diff
@@ -0,0 +1,9 @@
+pipeline:
+  eslint:
+    image: nextcloudci/eslint
```

Comment:
I see :-) But what if a new version of an image is tagged? Do you have to update all the `.drone.yml` files that use this image to point to the new version? :roll_eyes: